src/views/order_list_view.js

```diff
@@ -0,0 +1,113 @@
+//form will get rendered always
+// the bigger scope- captures the whole bottom part- the form AND the individual orders that will show up once the orders are created
+//
+import Backbone from 'backbone';
+// import _ from 'underscore';
+import QuoteView from '../views/quote_view';
+import Quote from '../models/quote';
+import TradeHistoryView from '../views/trade_history_view';
+import OrderView from '../views/order_view';
+import Order from '../models/order';
+
+const OrderListView = Backbone.View.extend({
+  initialize(params){
+    this.template = params.template;
+    this.bus = params.bus;
+    this.quotes = params.quotes;
+
+    this.listenTo(this.model, 'update', this.render);
+    this.listenTo(this.bus, 'quote_price_change', this.buySellQuote);
+  },
+  render() {
+    this.$('#orders').empty();
+
+    this.model.each((order) => {
+      const orderView = new OrderView({
+        model: order,
+        template: this.template,
+        tagName: 'li',
+        className: 'order', // orders?
+        bus: this.bus,
+      });
+      this.$('#orders').prepend(orderView.render().$el);
+    });
+    this.newOrderFormRender();
+    return this;
+  },
+  newOrderFormRender() {
+    this.$('#add-order-form select').html('');
+    this.quotes.each((quote) =>{
+      const symbol = quote.get('symbol');
+      this.$('#add-order-form select').append(`<option value="${symbol}">${symbol}</option>`);
+    });
+  },
+  events: {
+    'click button.btn-buy, button.btn-sell': 'addOrder',
+  },
+  addOrder(event) {
+    event.preventDefault();
+    const formData = this.getFormData();
+    console.log(formData);
+    const quotePrice = this.quotes.where({symbol: formData['symbol']})[0].get('price');
+    if (event.target.classList.contains('btn-buy')) {
+      formData['buy'] = true;
+      if (quotePrice <= formData['targetPrice']) {
+        // console.log('targetPrice must be lower than current price');
+        this.updateUser('Sorry! You must enter a number that is lower than the current price ');
```

Comment:
This functionality would be a great thing to add to the `Order` validations. The order would need a reference to the corresponding `Quote`, but you already find that when you look up the `quotePrice` above.